tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/trusts/ofstedPage.ts

```diff
@@ -142,6 +166,119 @@ class OfstedPage {
         this.elements.currentRatings.NoDataMessage().should('be.visible');
         return this;
     }
+
+    ///previous ratings///
+
+    public checkOfstedPreviousRatingsPageHeaderPresent(): this {
+        this.elements.subpageHeader().should('contain', 'Previous ratings');
+        return this;
+    }
+
+    public checkOfstedPreviousRatingsTableHeadersPresent(): this {
+        this.elements.previousRatings.SchoolNameHeader().should('be.visible');
+        this.elements.previousRatings.qualityOfEducationHeader().should('be.visible');
+        this.elements.previousRatings.behaviourAndAttitudesHeader().should('be.visible');
+        this.elements.previousRatings.personalDevelopmentHeader().should('be.visible');
+        this.elements.previousRatings.leadershipAndManagementHeader().should('be.visible');
+        this.elements.previousRatings.earlyYearsProvisionHeader().should('be.visible');
+        this.elements.previousRatings.sixthFormProvisionHeader().should('be.visible');
+        this.elements.previousRatings.beforeOrAfterJoiningHeader().scrollIntoView();
+        this.elements.previousRatings.beforeOrAfterJoiningHeader().should('be.visible');
+        return this;
+    }
+
+    public checkOfstedPreviousRatingsSorting(): this {
```

Comment:
This concerns me, I know it was used for testing the old Ofsted table sorting but I wonder whether it's actually testing anything at all...
The reason for my concern is that these Ofsted values are not sorted alphabetically, they're sorted by grade value so I would expect this page model to define what the correct sorting (`Outstanding` then `Good` then `Requires improvement` etc) looks like. I think we should review how the table sorting works after this story goes in.

Reply:
Sure thing - we can take a stab at a refactor for this after the Ofsted stuff is in.

It may be worth making a devops ticket to handle this?